# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lm_sensors/no_install_i2c-dev.h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./kernel/include/Module.mk.orig	2003-07-31 23:36:24.000000000 -0400
+++ ./kernel/include/Module.mk	2003-07-31 23:37:10.000000000 -0400
@@ -21,7 +21,7 @@
 MODULE_DIR := kernel/include
 KERNELINCLUDEDIR := $(MODULE_DIR)
 
-KERNELINCLUDEFILES := $(MODULE_DIR)/i2c-dev.h $(MODULE_DIR)/sensors.h
+KERNELINCLUDEFILES := $(MODULE_DIR)/sensors.h
 
 $(KERNELINCLUDEDIR)/sensors.h: $(KERNELINCLUDEDIR)/sensors.h.template
 	cat $@.template > $@
